modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts

```diff
@@ -232,6 +232,7 @@ declare module Liferay {
 	namespace ThemeDisplay {
 		export function getDefaultLanguageId(): string;
 		export function getLanguageId(): Language.Locale;
+		export function getSiteGroupId(): string | number;
```

Comment:
According https://cs.github.com/liferay/liferay-portal/blob/8383cb110656f21550f16ad96b56e21c5a28b66e/portal-kernel/src/com/liferay/portal/kernel/util/Portal.java#L1016 it should be just a number. Why you added `string`? 🥺

Reply:
Not sure, probably just added it in the heat of the moment to get things working lol